### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by ghost**
This PR focuses on making things cleaner. I am still working on it, some stuff is (unfortunately) broken. Fixing broken things is fun though!

### CODE REVIEW

**Comment by ghost** [comment]
!!!! NOTE !!!!: I don't plan on solving #8 with this PR. I'm just cleaning up the files to make sure everything's sorted and not bloated and just overall better.

**Comment by ghost** [comment]
I'm done now! @Rexogamer @Devnol I'll let you guys choose to merge it or not. Works on my machine, atleast.

File: debia.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+                            <li><a href="project-tangerine.html">Debia</a></li>
+                            <li><a href="meetthedevs.html">Meet the Developers</a></li>
+                        </ul>
+                    </nav>
+                </div>
+            </div>
+        </header>
+        <h1>Debia</h1>
+        <slogan>The multi-purpose bot you've always wanted... rewritten.</slogan>
+        <p>Debia is a complete rewrite of Cytrus-RE, but it's not made in JS: It's made in TypeScript.</p>
```
Comment (Rexogamer) [inline debia.html]:
```suggestion
        <p>Debia is a complete rewrite of Cytrus-RE, but it's not made in JS, it's made in TypeScript.</p>
```

**Comment by ghost** [comment]
By the way, if you want descs, you can do like me:

git commit -s -m "commit title" -m "commit desc"

You don't need the -s, but I use it because it's, well, formal.

File: dev/commandblock6417.html [hunk 2]
```diff
@@ -1,40 +1,10 @@
...
-        <br />He likes to fiddle with software and hardware but usually breaks things.
-        <br />He uses Mac OS Big Sur and likes to code in C, python, Javascript, albeit he's not that good at any of them.
-    	<br />He sometimes plays retro games like the boomer he is or he plays games on his Nintendo Switch. He doesn't play games a lot though.
-    	<br />He recently joined Terrific Tea Studios as he is interested in learning more JavaScript and html.
-    	<br />He is always trying to improve himself, while he sometimes acts like a tard.</h3>
-
-        <a href="https://github.com/Commandblock6417">GitHub Profile</a>
-        <a href="https://steamcommunity.com/id/0017534">Steam profile</a>
-        <a href="https://twitter.com/Commandblock641">Twitter profile</a>
+        <h1> Seems like you have stumbled upon an obsolete site, and your browser doesn't support redirections! Please click <a href="https://terrific-tea.github.io/team/Devnol.html">here to get to the new site.</a></h1>
```
Comment (Rexogamer) [inline dev/commandblock6417.html]:
```suggestion
        <h1> Seems like you have stumbled upon an obsolete page, and your browser doesn't support redirections! Please click <a href="https://terrific-tea.github.io/team/Devnol.html">here to get to the new site.</a></h1>
```

File: dev/odyssey346.html [hunk 3]
```diff
@@ -1,36 +1,10 @@
...
-        </header>
-        <h1>Odyssey346's bio</h1>
-        <h3>Odyssey346 is just your not-so-average Norwegian kid. 
-        <br />He usually messes around and doesn't really do anything other than play Team Fortress 2.
-        <br />He uses Visual Studio Code and likes using Linux, and made this site with the (obvious) help of the other devs.</h3>
-
-        <a href="https://github.com/Oxygemo">GitHub Profile</a>
-        <a href="https://steamcommunity.com/id/Odyssey346">Steam profile</a>
-        <a href="https://twitter.com/odyssey346">Twitter profile</a>
+        <h1> Seems like you have stumbled upon an obsolete site, and your browser doesn't support redirections! Please click <a href="https://terrific-tea.github.io/team/Odyssey346.html">here to get to the new site.</a></h1>
```
Comment (Rexogamer) [inline dev/odyssey346.html]:
```suggestion
        <h1> Seems like you have stumbled upon an obsolete page, and your browser doesn't support redirections! Please click <a href="https://terrific-tea.github.io/team/Odyssey346.html">here to get to the new site.</a></h1>
```

File: meetthedevs.html [hunk 4]
```diff
@@ -18,14 +18,16 @@
...
-                            <li><a href="project-tangerine.html">Project Tangerine</a></li>
+                            <li><a href="debia.html">Project Tangerine</a></li>
                             <li><a href="meetthedevs.html">Meet the Developers</a></li>
                         </ul>
                     </nav>
                 </div>
             </div>
         </header>
-        <h1>In this webpage you can read about the developers behind the Terrific Tea Studios team.</h1>
+        <h1>This site is actually obsolete, we've moved these to TTS' page instead (to not have to update both sites at once). Click any button to be sent to the (actual) bios!</h1>
```
Comment (Rexogamer) [inline meetthedevs.html]:
```suggestion
        <h1>This part of the site is actually obsolete as we've moved these to TTS' site instead (to not have to update both sites at once). Click any button to be sent to the (actual) bios!</h1>
```

File: meetthedevs.html [hunk 5]
```diff
@@ -18,14 +18,16 @@
...
+        <p>Or alternatively just go to the new bio page where there's more members and it's cleanier.</p>
```
Comment (Rexogamer) [inline meetthedevs.html]:
```suggestion
        <p>Or alternatively just go to the new bio page where there's more members and it's cleaner.</p>
```

**Review by Rexogamer — CHANGES_REQUESTED** [review]
Almost there :)

**Comment by ghost** [comment]
these are great suggestions, thanks for noticing! I'll get right to work.

**Comment by ghost** [comment]
Done. Commiting changes right now

**Comment by ghost** [comment]
Testing my email signature

----------------------------------
Alexander/Odyssey346
-----------------------------------

**Comment by ghost** [comment]
Trying again

On Fri, Dec 11, 2020 at 8:06 PM Odyssey346 <alej0hio2007@gmail.com> wrote:

>
> Testing my email signature
>
> ----------------------------------
> Alexander/Odyssey346
> -----------------------------------
>


-- 
Alexander/Odyssey346

Sorta dev @ https://terrific-tea.github.io

**Review by ghost — COMMENTED** [review]
lgtm go merge pls

**Review by Rexogamer — APPROVED** [review]

**Pull request merged**